nav2_route/graphs/aws_graph.geojson

```diff
@@ -1,132 +1,1664 @@
 {
-"type": "FeatureCollection",
```

Comment:
Just checking: did you update anything here or is it just reformatted from opening it in your editor?

Reply:
I am not sure what happened here, because I am not aware of changing this file. I have restored it from `main` branch.